src/applications/caregivers/config/chapters/signAsRepresentative/signAsRepresentativeYesNo.js

```diff
@@ -20,7 +20,7 @@ const representativePage = {
         labels: {
           yes: 'Yes. I want to upload my document now.',
           noRep: "Yes. But I'll provide my document later.",
-          no: "No. I don't have this document",
+          no: "No. I don't have this document.",
```

Comment:
Content recommends we use typographical apotrophes (`’` or `\u2019`).

Reply:
Good to know. I went ahead and updated other instances of this. [Commit](https://github.com/department-of-veterans-affairs/vets-website/pull/17186/commits/8f4660b20b9223f39ac69241ce0bbefc567c9d30)